### PR TITLE
pid configuration and reflection

### PR DESCRIPTION
**Pull request by adamdbrw**
Signed-off-by: Adam Dabrowski <adam.dabrowski@robotec.ai>

Note - not a final solution, but an enabler

Final solution - use abstraction of Drive Model

### CODE REVIEW

File: Code/Source/VehicleDynamics/VehicleModelComponent.cpp [hunk 1]
```diff
@@ -105,7 +112,7 @@ namespace VehicleDynamics
         }
 
         uint64_t deltaTimeNs = deltaTime * 1000000000;
```
Comment (michalpelka) [inline Code/Source/VehicleDynamics/VehicleModelComponent.cpp:114]:
```suggestion
        const uint64_t deltaTimeNs = deltaTime * 1e9;
```

**Review by michalpelka — APPROVED** [review]

File: Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp [hunk 2]
```diff
@@ -29,14 +59,14 @@ namespace VehicleDynamics
     // TODO - speed and steering handling is quite similar, possible to refactor?
     void SimplifiedDriveModel::ApplySteering(float steering, const ChassisConfiguration& vehicleChassis, uint64_t nsDt)
     {
-        const double nsDtSec = double(nsDt)/1e9;
```
Comment (pawelbudziszewski) [inline Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp:32]: We should leave this.

File: Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp [hunk 3]
```diff
@@ -50,7 +80,7 @@ namespace VehicleDynamics
                 continue;
             }
 
-            auto torque = pidCommand * nsDtSec;
```
Comment (pawelbudziszewski) [inline Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp:53]: We should leave this.

**Review by pawelbudziszewski — COMMENTED** [review]
Feature requests:

- Easy way to define max speed (the target speed when pressing up arrow) - eg. expose it to the UI
- Easy way to define wheel rotation axis

**Pull request merged**